src/controllers/import.js

```diff
@@ -32,6 +32,25 @@ import { ImportJobDto } from '../dto/import-job.js';
  * @constructor
  */
 function ImportController(context) {
+  /**
+   * The import controller has a number of scopes that are required to access different parts of the
+   * import functionality. These scopes are used to validate the authenticated user has the required
+   * level of access.
+   * The scopes are:
+   * READ: 'imports.read' - allows users to view import jobs
```

Comment:
Would it be easier to just add a comment next to each entry? This way it cannot get out of sync with the comment.